### DISCUSSION TITLE
Link to konstin2/maturin dockerfile

### DISCUSSION THREAD

**Discussion by Keats** [PyO3/maturin#715] [Q&A] [answered]
Hey,

I can't find the Dockerfile used in that image, is it tracked in any repo? There's https://hub.docker.com/r/konstin2/maturin/dockerfile but I don't think it's the latest.

Thanks.

**Comment by messense** [comment, marked as answer]
https://github.com/PyO3/maturin/blob/main/Dockerfile

**Comment by Keats** [comment]
🤦 thanks
